js/index.js

```diff
@@ -32,4 +32,21 @@ document.addEventListener('DOMContentLoaded', e => {
     document.querySelector('.intro-background').addEventListener('transitionend', function() {this.style.display = 'none';});
     Particles.init(config);
   });
-});
\ No newline at end of file
+
+  document.querySelector('.mute-audio').addEventListener('click', e => {
+    // mute/unmute toggle
+    var element = document.querySelector('.mute-audio');
```

Comment:
Don't use an arrow function here. Then "this" will automatically point to the button.

Reply:
Changed from arrow function to vanilla JS function.